plugins/picture-in-picture/front.js

```diff
@@ -75,19 +92,22 @@ const listenForToggle = () => {
 }
 
 function observeMenu(options) {
+	useNativePiP = options.useNativePiP;
 	document.addEventListener(
 		"apiLoaded",
 		() => {
 			listenForToggle();
 			// remove native listeners
-			cloneButton(".player-minimize-button").onclick = () =>  {
-				global.togglePictureInPicture();
-				setTimeout(() => $('#player').click());
+			cloneButton(".player-minimize-button").onclick = async () => {
+				const isUsingNativePiP = await global.togglePictureInPicture();
+				if (!isUsingNativePiP) {
+					setTimeout(() => $("#player").click());
+				}
```

Comment:
Why `if (!isUsingNativePiP)`? For me, the video pause when clicking the PiP button, regardless if it's using the native one or not.

```suggestion
				setTimeout(() => $("#player").click());
```